app/graphql/types/git/commit_type.rb

```diff
@@ -94,9 +94,19 @@
     end
 
     resolve(lambda do |commit, arguments, _context|
-      GitFile.new(commit,
-                  arguments['path'],
-                  load_all_data: arguments['loadAllData'])
+      git_file = GitFile.new(commit,
+                             arguments['path'],
+                             load_all_data: arguments['loadAllData'])
+      git_file if git_file.exist?
+    end)
+  end
+
+  field :lsFiles, !types[!types.String] do
```

Comment:
We could do it without blowing up the computation time if we only query for `name` and `path`. However, it would be a blob lookup (including loading up to 10 MB of the data into memory per blob) for every file of the repository (some ontology repositories have many files) as soon as we query for more than just path and name.

I vote for not doing this.